utils/Constants.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+var Constants map[string]string = map[string]string{
+	"ENV":           "ENVIRONMENT",
+	"DEV":           "DEVELOPMENT",
+	"PROD":          "PRODUCTION",
+	"REGION":        "AWS_REGION",
+	"ACCESS_KEY":    "AWS_ACCESS_KEY",
+	"SECRET_KEY":    "AWS_SECRET_KEY",
+	"FF_TABLE_NAME": "FEATURE_FLAG_TABLE_NAME",
+}
+
+var FeatureFlagStatus map[string]string = map[string]string{
+	"ENABLED":  "ENABLED",
+	"DISABLED": "DISABLED",
+}
+
+var FeatureFlagAttributes map[string]string = map[string]string{
```

Comment:
if key and values are same, do you really need a map ?

Reply:
Wanted to have an `Enum` kind of behaviour, instead of directly using strings.
Was thinking to use something like this 
``` 
const (
    Monday    = "Monday"
    Tuesday   = "Tuesday"
    Wednesday = "Wednesday"
    Thursday  = "Thursday"
    Friday    = "Friday"
    Saturday  = "Saturday"
    Sunday    = "Sunday"
)
```

what are your thoughts on this @dev-gv

Reply:
yeah looks good. `UpperCamelCase` if you want to export them outside the package. `lowerCamelCase` if they are meant to be internally used.

https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps